### ISSUE TITLE
IllegalStateException in ClassLevelValidator

### ISSUE THREAD

**Issue by amischler** [dooApp/FXForm2#150]
```
[DEBUG]	2017-08-25 14:40:51,499 c.d.d.c.DsdkApplication{JavaFX Application Thread} - Called endChange before beginChange 
java.lang.IllegalStateException: Called endChange before beginChange
	at javafx.collections.ListChangeBuilder.endChange(ListChangeBuilder.java:538) ~[jfxrt.jar:na]
	at javafx.collections.ObservableListBase.endChange(ObservableListBase.java:205) ~[jfxrt.jar:na]
	at com.sun.javafx.collections.ObservableListWrapper.clear(ObservableListWrapper.java:157) ~[jfxrt.jar:na]
	at javafx.beans.binding.ListExpression.clear(ListExpression.java:293) ~[jfxrt.jar:na]
	at com.dooapp.fxform.validation.ClassLevelValidator.validate(ClassLevelValidator.java:62) ~[dsdk-launcher.jar:na]
	at com.dooapp.fxform.controller.PropertyEditorController$2.changed(PropertyEditorController.java:100) ~[dsdk-launcher.jar:na]
	at com.sun.javafx.binding.ExpressionHelper$Generic.fireValueChangedEvent(ExpressionHelper.java:361) ~[jfxrt.jar:na]
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81) [jfxrt.jar:na]
	at javafx.beans.property.DoublePropertyBase.fireValueChangedEvent(DoublePropertyBase.java:106) [jfxrt.jar:na]
	at javafx.beans.property.DoublePropertyBase.markInvalid(DoublePropertyBase.java:113) [jfxrt.jar:na]
	at javafx.beans.property.DoublePropertyBase.set(DoublePropertyBase.java:147) [jfxrt.jar:na]
```

**Comment by amischler** [comment]
Deployed in 8.2.0-SNAPSHOT

**Comment by Rizen59** [comment]
Fixed in 8.1.3
